pkg/gen/input/workflows/internal/file/update_chart.yaml.template

```diff
@@ -0,0 +1,99 @@
+{{{{ .Header }}}}
+name: Create Update PR
```

Comment:
Optional!

I think adding a two or three sentence explanationatory comment at the head of the file would help people stumbling on this file in app repositories.